src/ToolBox/superpmi/superpmi/commandline.cpp

```diff
@@ -119,11 +119,17 @@ void CommandLine::DumpHelp(const char* program)
     printf("     Ignored: neither MSVCDIS nor CoreDisTools is available.\n");
 #endif // USE_COREDISTOOLS
     printf("\n");
+    printf(" -forcejitoption key=value\n");
```

Comment:
An alternative here would be to expand the grammar for `jit{,2}option` to `-jitoption [force] key=value`. That should be easy enough to implement, and might make the options a bit more straightforward.

Reply:
Done, please take a look.